pkg/aws/policies.go

```diff
@@ -284,9 +294,12 @@ func (c *awsClient) EnsurePolicy(policyArn string, document string,
 
 	policyArn = aws.StringValue(output.Policy.Arn)
 
-	isCompatible, err := c.IsPolicyCompatible(policyArn, version)
-	if err != nil {
-		return policyArn, err
+	isCompatible := false
```

Comment:
Should this be true for `force==true`?

Reply:
No we are always considering incompatible when forced so it will go through recreating policies